bears/configfiles/DockerfileLintBear.py

```diff
@@ -22,6 +22,8 @@ class DockerfileLintBear:
     AUTHORS_EMAILS = {'coala-devel@googlegroups.com'}
     LICENSE = 'AGPL-3.0'
     CAN_DETECT = {'Syntax', 'Smell'}
+    # Ignore PycodestyleBear (E501)
+    SEE_MORE = 'https://github.com/projectatomic/dockerfile_lint#dockerfile-lint'
```

Comment:
Hmm although this is described in the issue, I would like to see this in a new commit. I would file a new issue explicitly for this and gonna assign you, so you would effectively solve 2 issues. Is that okay with you? :)
(I would also adapt the old issue to exclude the `SEE_MORE` change, so everything is properly separated.)

Reply:
So we have one commit fixing the grammatical issues and one for the `SEE_MORE` attribute

Reply:
That is fine by me! Or you can leave it the second issue open for newcomers since there are many newcomers looking for `difficulty/newcomer` task and I have already done another newcomer task for `corobo`. I will pick it up if there are no takers within a couple of days.